src/main/resources/mappers/export/CnaSegmentMapper.xml

```diff
@@ -5,45 +5,43 @@
     <select id="getCnaSegments" resultType="org.cbioportal.application.file.model.CnaSegment"
             resultSetType="FORWARD_ONLY">
         SELECT
-            cns.SEG_ID,
-            s.STABLE_ID AS sampleId,
-            cns.chr as chr,
-            cns.start as start,
-            cns.end as end,
-            cns.num_probes as numProbes,
-            cns.segment_mean as segmentMean
+            cns.seg_id,
+            s.stable_id AS sampleId,
+            cns.chr AS chr,
+            cns.start AS start,
+            cns.end AS end,
+            cns.num_probes AS numProbes,
+            cns.segment_mean AS segmentMean
         FROM copy_number_seg cns
-        JOIN cancer_study cs ON cs.CANCER_STUDY_ID = cns.CANCER_STUDY_ID
-        JOIN sample s ON s.INTERNAL_ID = cns.SAMPLE_ID
+        JOIN cancer_study cs ON cs.cancer_study_id = cns.cancer_study_id
+        JOIN sample s ON s.internal_id = cns.sample_id
         <if test="sampleIds != null">
             JOIN (
                 SELECT *
                 FROM (VALUES
                 <foreach item="sid" collection="sampleIds" separator=",">
-                    ROW(#{sid})
+                    (#{sid})
```

Comment:
@forus just a heads up that in Clickhouse scenario, using foreach on very large lists (e.g. sample ids) can be problematic.  Yes, foreach is executed in java prior to submission to clickhouse, however, it has to do with the way prepared statements work differently in CL than MySQL.   If you search the mappers for ```apache.ibatis.type.ArrayTypeHandler``` you'll see an alternative style of array handling which is much more performant.